examples/templates/README.md

```diff
@@ -1,21 +1,45 @@
 ## Templates Guidline
 
-These templates are a base for the new code examples.
+This folder contains templates for new code examples. Every new code example should be made using these templates. 
+
+#### Prerequisite
+To work with code examples (and Handsontable development in general) use at least Node version `15.9.0`. If you have Node Version Manager installed, you can just type in the terminal `nvm use`.
 
 ### Creating a new example
 
-All new code examples should be added to the `/examples/next` directory. Based on them you can create versioned code examples.
+In this guideline, I will walk you through the process of creating new code examples. 
+
+> If you need some inspiration, what can be done with code examples, go ahead see existing examples in `examples/next/docs/` to get inspired!
+
+Firstly, create a new git branch because you will need to commit new examples to the Handsontable repo.
+
+Currently, we are inside the `examples/templates` directory where all templates for future code examples live. You can see that we have templates for examples made in React, Angular, Vue, and of course vanilla Javascript.
```

Comment:
Not sure what you meant by this "Currently, we are inside the `examples/templates` directory". Maybe sth like "We store all code examples in the examples/templates directory in the main repository. We use them in many different places, such as our blog, documentation, and more."

Reply:
> Not sure what you meant by this "Currently, we are inside the examples/templates directory".

You're right, the intention of this sentence isn't clear. 

> Maybe sth like "We store all code examples in the examples/templates directory in the main repository. We use them in many different places, such as our blog, documentation, and more."

But that's not true. We store all code examples in `next/` and later inside versioned examples like `8.4.0/` etc. In `examples/templates/` we store "templates" for new examples.